.github/workflows/build.yml

```diff
@@ -20,48 +20,36 @@ jobs:
     # when it comes to building docker images.
     runs-on: ubuntu-latest
     steps:
-      -
-        name: Checkout repository
+      - name: Checkout repository
         uses: actions/checkout@v4
-      -
-        name: Set up docker buildx
+      - name: Set up docker buildx
         uses: docker/setup-buildx-action@v3
-      -
-        name: Login to DockerHub
-        uses: docker/login-action@v3
-        with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
-      -
-        name: Login to Github Container Registry
+      - name: Login to Github Container Registry
         uses: docker/login-action@v3
         with:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      -
-        name: Build and push
+      - name: Build and push
         uses: docker/build-push-action@v5
         with:
           context: .
           platforms: linux/amd64,linux/arm64
           push: ${{ github.event_name != 'pull_request' }}
           build-args: |
-            BASE_IMAGE_NAME=${{ matrix.base_image_name }}
-            BASE_IMAGE_TAG=${{ matrix.base_image_tag }}
+            BASE_IMAGE=${{ matrix.base_image_name }}:${{ matrix.base_image_tag }}
```

Comment:
We have `BASE_IMAGE_NAME` and `BASE_IMAGE_TAG` instead of `BASE_IMAGE` now: https://github.com/ros-tooling/setup-ros-docker/blob/43975566425de6c72b361565cdc663e5f222a52f/Dockerfile#L12-L18